openwisp_network_topology/admin.py

```diff
@@ -3,20 +3,39 @@
                                             AbstractNodeAdmin,
                                             AbstractTopologyAdmin)
 
-from .base.admin import MultitenantAdminMixin
+from .base.admin import (MultitenantAdminMixin,
+                         MultitenantObjectFilter,
+                         MultitenantOrgFilter)
 from .models import Link, Node, Topology
 
 
 class TopologyAdmin(MultitenantAdminMixin, AbstractTopologyAdmin):
     model = Topology
 
 
+TopologyAdmin.list_display += ('organization',)
+TopologyAdmin.list_filter += (('organization', MultitenantOrgFilter),)
+TopologyAdmin.fields.insert(1, 'organization')
+
+
 class NodeAdmin(MultitenantAdminMixin, AbstractNodeAdmin):
     model = Node
+    list_display = ['name', 'organization', 'topology', 'addresses']
+    list_filter = [('organization', MultitenantOrgFilter),
+                   ('topology', MultitenantObjectFilter)]
```

Comment:
I see, at glance it seems it can't work because it requires some extra work. You may create a specific issue for this task, remove the code from this PR and go ahead. We'll add this feature later.